backend/src/main/java/gov/cdc/usds/simplereport/SimpleReportApplication.java

```diff
@@ -9,9 +9,11 @@
 import org.springframework.orm.jpa.support.OpenEntityManagerInViewFilter;
 
 import gov.cdc.usds.simplereport.config.InitialSetupProperties;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
 @EnableConfigurationProperties(InitialSetupProperties.class)
+@EnableScheduling
```

Comment:
When we get past step 1, this will want to move to a separate `@Configuration` class (which can also handle any property wiring for the schedule), which can be given a separate bean profile—otherwise we'll be running scheduled uploads in local dev instances, and nobody wants that.

Reply:
Good thing to remember. I was thinking that the cron job could still run in dev, but the @Configuration would have some value that would cause the function to return instead running.